src/common/transformations/src/transformations/symbolic_transformations/symbolic_optimizations.cpp

```diff
@@ -206,13 +206,25 @@ ov::pass::SymbolicOptimizations::SymbolicOptimizations(bool full_run,
 bool ov::pass::SymbolicOptimizations::run_on_model(const std::shared_ptr<ov::Model>& m) {
     RUN_ON_FUNCTION_SCOPE(SymbolicOptimizations);
 
-    // Eliminate Squeeze/Unsqueeze might convert Squeeze/Unsqueeze ops to Reshape
-    // it may break NNCF patterns and lead to unexpected FakeQuantize ops in the model.
-    // So we decided to disable these passes in SymbolicOptimizations.
     const auto& pass_config = m_manager->get_pass_config();
-    pass_config->disable<EliminateSqueeze>();
-    pass_config->disable<EliminateUnsqueeze>();
+    
+    // If PassConfig is NOT shared (use_count == 1), we can safely modify it
+    // If it IS shared (use_count > 1), we should not modify it to avoid affecting other passes
+    if (pass_config.use_count() == 1) {
```

Comment:
This solution is based on the fact that `SymbolicOptimizations` in MOC pipeline will never pass pass config as `SymbolicOptimizations` parameter. If this becomes necessary, and the config is passed as a parameter, NNCF pipeline will be broken (as it is described in the original comment)

Please consider alternative solution: always disable `EliminateSqueeze`/`EliminateUnsqueeze` passes, but enable them back after `m_manager->run_passes(m);` call (but only if they were enabled before disabling in the `SymbolicOptimizations`: we can check it easily using pass config API). This will preserve `SymbolicOptimizations` behavior and will avoid destructive for the rest pipeline shared `pass_config` modification

Reply:
I fixed another way - moved into the main cpu pipeline